tests/testsuite/cargo_remove/invalid_gc_patch/out/Cargo.toml

```diff
@@ -3,3 +3,6 @@
 [workspace]
 members = ["serde", "serde_derive"]
 
+[patch.crates-io]
```

Comment:
I assume this is tied into the previous commit. Could you squash this in so each commit stands on its own with passing tests?

Reply:
Will do. Is the goal to make it so every commit has no failing tests? I actually don't understand why. I read about bisection but it's not clear how that comes into play on a branch. Does it have to do with how the branch is merged in?

Reply:
Ideally, we don't squash on merge but preserve the commits.

Reply:
Also, it makes it clearer to the review when they can see all of the related changes in one commit to know what is related

Reply:
Ah ok that makes sense now